### ISSUE TITLE
question: [Quantization] Which files to change to make inference faster for Q8BERT?

### ISSUE THREAD

**Issue by sarthaklangde** [IntelLabs/nlp-architect#221]
I know from previous issues it is mentioned that that Q8BERT was just an experiment to measure the accuracy of quantized BERT model. But, given that the performance is good, what changed would need to be made to torch.nn.quantization file to replace the FP32 operations by INT8 operations?

Replacing the FP32 Linear layers with the torch.nn.quantized.Linear should theoretically work since it will have optimized operations, but it doesn't. Same for other layers.

If someone could just point out how to improve the inference speed (hints, tips, directions, code, anything), it would be helpful since the model is performing really well. I don't mind even creating a PR once those changes are done so that it merges with the main repo.

Thank you!

**Comment by Ayyub29** [comment]
Do you find the answer for this?
